js/startposition.js

```diff
@@ -45,6 +45,13 @@ function Startposition(map)
 		else
 			self.setPosition();
 	};
+
+	// add marker
+	var marker = L.marker([params['lat'], params['lon']], {
+		src: "https://openrailwaymap.org/js/images/marker-icon.png"
+	});
+	marker.addTo(this.map);
+	
```

Comment:
This fails if the parameters are not part of the query string (unset) and lead to a JavaScript error. If we accept this pull request, your inserted sections needs to moved behind the call of `this.map.setView`.

Reply:
The insertion of the proposed changes in startposition.js I did at the wrong position, sorry. Will be changed immediately.

But in my opinion the additional <code class="notranslate">marker=true</code> has to be assigned on the serverside as part of the query string to the dictionary <code class="notranslate">params</code>. Is it right or have I to do further changes?